src/engine-actions/get-max-channel-for-address.js

```diff
@@ -0,0 +1,35 @@
+const { Big } = require('../utils')
+const getChannelsForRemoteAddress = require('./get-channels-for-remote-address')
+
+/**
+ * Get maximum balance from all channels (inbound or outbound) for a given address
+ * @param {string} address
+ * @param {Object} [options={}]
+ * @param {boolean} [options.outbound=true] - outbound is true if checking outbound channels, false if inbound
+ * @returns {Object} res
+ * @returns {string} res.maxBalance - the max balance in all open channels.
+ */
+async function getMaxChannelForAddress (address, { outbound = true } = {}) {
```

Comment:
`getMaxChannelBalanceForAddress` and only return the balance in this method. Since its only one value for this function, we dont need to return an object.

It looks like the relayer doesn't have any other uses for the old method `getMaxChannel` so we should be OK to change the API for this new function

Reply:
I retract my comment. @martineehrlich recommended we wait for the change and that sounds right. The reason why `getMaxChannelForAddress` and `getMaxChannel` returns an empty object, is so that if NO channels exist, we'll receive an undefined value in the code.